pages/formester-vs-paperform.vue

```diff
@@ -475,3 +538,13 @@ thead th {
   }
 }
 </style>
+<style>
+.present{
+  color: #6BC8AD;
+  font-weight: 600
+}
+.absent{
+  color: #F35E54; 
+  font-weight: 600
+}
+</style>
```

Comment:
can be skipped as these classes aren't scoped and are added on other pages too, for example - `pages/formester-vs-jotform.vue`